### PR TITLE
Awesome Books Project

### PR DESCRIPTION
**Pull request by TekyRey**
### **Awesome Books Project**
In this project we used plain JavaScript and html to:

- Create a html form that accepts book tittle and book author
- Store book data as objects in an array
- Add delete and add functionality
- display newly added books

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Conference Page
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly update the title with the current project for a clear understanding of the topic.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,41 @@
...
 
+> Book Records app
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly add a description of the project for a clear understanding of what the project is about.

File: README.md [hunk 3]
```diff
@@ -1,56 +1,41 @@
...
+👤 **Rehema Mwaka**
+
+- GitHub: [@githubhandle](https://github.com/TekyRey)
+- Twitter: [@twitterhandle](https://twitter.com/RehemaMwaka1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/rehema-mwaka-48a1801ab)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Henry Karamaga**
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly add the contact details of your collaborator.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @TekyRey & @henrykgggg, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Good PR Title and Description
✔️ All linters passing

## Required 
- [ ] Update Readme - _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @TekyRey,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**